src/platform/silabs/efr32/BLEChannelImpl.cpp

```diff
@@ -27,16 +27,25 @@ namespace Internal {
 
 namespace {
 
+inline constexpr uint8_t kAdvertisingFlagStaticRandomAddress = 1;
+inline constexpr uint8_t kAttErrorCodeNoError                = 0;
+inline constexpr uint8_t kAttErrorCodeInvalidHandle          = 0x01;
+inline constexpr uint8_t kAttErrorApplicationError           = 0x80;
+// For ble transmit, the size be smaller than  SL_BGAPI_MAX_PAYLOAD_SIZE - SL_BGAPI_MSG_HEADER_LEN : 256 - 5 = (251)
+// For sl_bt_gattdb_add_uuid128_characteristic, the size must be smaller than 255 - 30, so we pick a value below those thresholds.
+inline constexpr uint8_t kCharacteristicBufferSize = 220;
+inline constexpr uint8_t kIndicateTimeout          = 30;
+
 // Side Channel UUIDS
-const uint8_t kSideServiceUUID[16] = { 0x01, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0x00, 0x80,
-                                       0x00, 0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
-const uuid_128 kRxUUID = { .data = { 0x01, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0xAD, 0xDE, 0xEF, 0xBE, 0xAD, 0xDE, 0x00, 0xEE, 0xFF,
-                                     0xC0 } };
-const uuid_128 kTxUUID = { .data = { 0x02, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0xAD, 0xDE, 0xEF, 0xBE, 0xAD, 0xDE, 0x00, 0xEE, 0xFF,
-                                     0xC0 } };
+constexpr uint8_t kSideServiceUUID[16] = { 0x01, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0x00, 0x80,
+                                           0x00, 0x10, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+constexpr uuid_128 kRxUUID = { .data = { 0x01, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0xAD, 0xDE, 0xEF, 0xBE, 0xAD, 0xDE, 0x00, 0xEE, 0xFF,
+                                         0xC0 } };
+constexpr uuid_128 kTxUUID = { .data = { 0x02, 0x00, 0x00, 0xEE, 0xFF, 0xC0, 0xAD, 0xDE, 0xEF, 0xBE, 0xAD, 0xDE, 0x00, 0xEE, 0xFF,
+                                         0xC0 } };
```

Comment:
There was discussion about configurability of those UUID.

Reply:
Yeah I see a config file in the future of the BLE Side channel... mind if we postpone it? I'd like the config to include the max connection and that needs to interact to the max connection in MatterOBLE and I would like to convert MatterOBLE to a channel before I handle the connection sharing.